web/src/components/DandisetsPage.vue

```diff
@@ -55,48 +76,86 @@ export default {
       type: String,
       required: true,
     },
-    module: {
-      type: String,
-      required: true,
+    user: {
+      type: Boolean,
+      required: false,
+      default: false,
     },
   },
   data() {
     return {
-      selection: 'newest',
-      options: basicDandisetSortingOptions,
+      sortingOptions,
+      sortOption: Number(this.$route.query.sortOption) || 0,
+      sortDir: this.$route.query.sortDir || 1,
+      page: Number(this.$route.query.page) || 1,
+      total: 0,
     };
   },
   computed: {
-    page: {
-      get() {
-        return this.moduleRoot.page;
-      },
-      set(page) {
-        this.changePage({ page });
-      },
+    listingUrl() {
```

Comment:
Can you change this to a prop? I think this should be encoded in `MyDandisetsView`, `PublicDandisetsView` et al.

Reply:
I actually don't like the idea of arbitrarily passing the URL into the component as a prop, as I feel like we should be defining our use cases here, but if you think it'll make more sense when we add the search results page I could see it being less of a hassle

Reply:
My vague architectural intuition is that DandisetsPage contains things that are in common between all the listing pages. The URL is different for different pages, so it would be defined in the listing page.

That said, it occurs to me now that this might have to be reswizzled a bit to deal with search requests, since those would involve including a search query. I'm fine with what's here, with the caveat that I might have to reevaluate it soon.